File: Conscious Python - OOP/step 02. Атрибуты и методы/2.4/2.4.01.py
'''

Задание:

    Создайте класс Kitty.
    Создайте метод print_cat (не забудьте про self).
    Внутри метода создайте print('Hello, Kitty'). Метод выводит на экран строку 'Hello, Kitty'.
    Создайте экземпляр cat.
    Вызовите метод через экземпляр cat.метод(), чтобы вывести на экран текст. Функцию print использовать не нужно.

Sample Input:

Sample Output:

Hello, Kitty

'''

class Kitty:
    def print_cat(self):
        print('Hello, Kitty')
        
cat = Kitty()
cat.print_cat()

File: JavaScript for Beginners/2  Стандартные объекты/2.4.01.js
/*

В этом задании в нашу функцию testArray передаются два массива случайной длины заполненные случайными числами. 
Вам нужно сосчитать сумму всех элементов обоих массивов и возвратить ее из функции.

Sample Input:

[0, 1, 5, 3]  [3]

Sample Output:

12

*/

function testArray(a, b) {
    let sum = 0;
    a = a.concat(b);
    for (num of a) {
        sum+=num;
    }
    return sum;
}
